drivers/SmartThings/matter-thermostat/profiles/room-air-conditioner-humidity-fan-wind-heating-cooling.yml

```diff
@@ -1,4 +1,4 @@
-name: room-air-conditioner
+name: room-air-conditioner-humidity-fan-wind-heating-cooling
```

Comment:
We should have a version of this profile that is exactly the same, except without humidity

Reply:
we do have one like that- room-air-conditioner-humidity-fan-wind-heating-cooling

Reply:
Oh I see, could you change it so that the capabilities are listed in the same order? It looks like the `thermostatOperatingState` just needs to be moved below the setpoint capabilities like is done here. We'll want the capability order to be internally consistent between the profiles

Reply:
ah, good catch!